components/Stages/StagesList/StagesList.scss

```diff
@@ -0,0 +1,87 @@
+.stages-list {
+  display: flex;
+  flex-flow: column;
+  gap: 56px;
+
+  &__number {
+    position: relative;
+    z-index: 0;
```

Comment:
The default z-index has a value of 0
it seems that this property is superfluous